spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/neo4j/Neo4jHealthIndicator.java

```diff
@@ -39,7 +41,7 @@
 	/**
 	 * The Cypher statement used to verify Neo4j is up.
 	 */
-	static final String CYPHER = "match (n) return count(n) as nodes";
+	static String CYPHER = "match (n) return count(n) as nodes";
```

Comment:
If the `CYPER` is configurable this should stay final (`DEFAULT_CYPHER`) and you should have a regular `cypher` field that's initialized with the default